[2016-01-08 13:42:13,923] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:42:13,934] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsTrue(DefaultCommandTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:42:13,944] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsFalse(DefaultCommandTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:53:01,783] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 13:53:33,776] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:53:51,817] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 13:54:04,581] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testException(CommandTest.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:04,600] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.iface.CommandTest.testCommandWithResult(CommandTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:04,609] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultCommandAsChainFalse(CommandTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:04,633] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultBehaviorWithException(CommandTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:04,636] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.InjectionChainBuilder.executeAsChain(InjectionChainBuilder.java:57)
	at de.mwolff.command.chainbuilder.InjectionChainBuilderTest.testSpringChainBuilder(InjectionChainBuilderTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:04,921] ERROR Error while executing chain.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommandContainer.execute(DefaultCommandContainer.java:84)
	at de.mwolff.commons.command.DefaultCommandContainerTest.testChainWithError(DefaultCommandContainerTest.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:06,330] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:06,331] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsTrue(DefaultCommandTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:06,361] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsFalse(DefaultCommandTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
[2016-01-08 13:54:34,041] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:33,825] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsTrue(DefaultCommandTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:33,868] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsFalse(DefaultCommandTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:33,960] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.InjectionChainBuilder.executeAsChain(InjectionChainBuilder.java:57)
	at de.mwolff.command.chainbuilder.InjectionChainBuilderTest.testSpringChainBuilder(InjectionChainBuilderTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:33,983] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testException(CommandTest.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:33,998] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.iface.CommandTest.testCommandWithResult(CommandTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:34,005] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultCommandAsChainFalse(CommandTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:34,010] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultBehaviorWithException(CommandTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-08 14:01:34,027] ERROR Error while executing chain.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommandContainer.execute(DefaultCommandContainer.java:84)
	at de.mwolff.commons.command.DefaultCommandContainerTest.testChainWithError(DefaultCommandContainerTest.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:91)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:57)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,540] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testException(CommandTest.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,547] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.iface.CommandTest.testCommandWithResult(CommandTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,553] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultCommandAsChainFalse(CommandTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,554] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultBehaviorWithException(CommandTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,577] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.InjectionChainBuilder.executeAsChain(InjectionChainBuilder.java:57)
	at de.mwolff.command.chainbuilder.InjectionChainBuilderTest.testSpringChainBuilder(InjectionChainBuilderTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,587] ERROR Error while executing chain.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommandContainer.execute(DefaultCommandContainer.java:84)
	at de.mwolff.commons.command.DefaultCommandContainerTest.testChainWithError(DefaultCommandContainerTest.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,630] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsTrue(DefaultCommandTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,644] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsFalse(DefaultCommandTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:04:48,974] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,189] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testException(CommandTest.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,194] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.iface.CommandTest.testCommandWithResult(CommandTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,200] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultCommandAsChainFalse(CommandTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,203] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.iface.CommandTest.testDefaultBehaviorWithException(CommandTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,215] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.InjectionChainBuilder.executeAsChain(InjectionChainBuilder.java:57)
	at de.mwolff.command.chainbuilder.InjectionChainBuilderTest.testSpringChainBuilder(InjectionChainBuilderTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,221] ERROR Error while executing chain.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommandContainer.execute(DefaultCommandContainer.java:84)
	at de.mwolff.commons.command.DefaultCommandContainerTest.testChainWithError(DefaultCommandContainerTest.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,237] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:39)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.execute(SimpleTestCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.samplecommands.SimpleTestCommand.executeAsChain(SimpleTestCommand.java:50)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsTrue(DefaultCommandTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,243] INFO  Chain is aborted.
java.lang.NullPointerException
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:38)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandTest.executeAsChainReturnsFalse(DefaultCommandTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
[2016-01-12 20:05:57,416] INFO  Chain is aborted.
de.mwolff.commons.command.iface.CommandException: Method is not implemented yet.
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:39)
	at de.mwolff.commons.command.samplecommands.ExceptionCommand.execute(ExceptionCommand.java:1)
	at de.mwolff.commons.command.DefaultCommand.executeAsChain(DefaultCommand.java:77)
	at de.mwolff.commons.command.DefaultCommandContainer.executeAsChain(DefaultCommandContainer.java:99)
	at de.mwolff.command.chainbuilder.XMLChainBuilder.executeAsChain(XMLChainBuilder.java:33)
	at de.mwolff.command.chainbuilder.XMLChainBuilderTest.oneExceptionCommandInserted(XMLChainBuilderTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:168)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.infinitest.testrunner.JUnit4Runner.runJUnitTest(JUnit4Runner.java:97)
	at org.infinitest.testrunner.JUnit4Runner.runTest(JUnit4Runner.java:63)
	at org.infinitest.testrunner.TestRunnerProcess.runTest(TestRunnerProcess.java:70)
	at org.infinitest.testrunner.TestRunnerProcess.writeTestResultToOutputStream(TestRunnerProcess.java:112)
	at org.infinitest.testrunner.TestRunnerProcess.main(TestRunnerProcess.java:89)
